### PR TITLE
UPnP: add support for video's countries through xbmc:country

### PR DESCRIPTION
**Pull request by Montellese**
This adds support for an `xbmc:country` tag in the DIDL-Lite of a media item sent over UPnP.

### CODE REVIEW

File: lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaItem.cpp [hunk 1]
```diff
@@ -606,6 +607,16 @@ PLT_MediaObject::ToDidl(NPT_UInt64 mask, NPT_String& didl)
         didl += "</xbmc:uniqueidentifier>";
     }
 
+    // genre
```
Comment (Razzeee) [inline lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaItem.cpp]: Wrong comment?

Comment (Montellese) [inline lib/libUPnP/Platinum/Source/Devices/MediaServer/PltMediaItem.cpp]: Argh yup. Now I have to change the commit and the patch file :-/

**Comment by Montellese** [comment]
Fixed the comment. jenkins build and merge

**Comment by Montellese** [comment]
jenkins build and merge

**Comment by kfarhat** [comment]
Please stop email me

**Pull request merged**